test/e2e/v1beta1/scripts/setup-katib.sh

```diff
@@ -76,6 +76,13 @@ if [ $? -ne 1 ]; then
   exit 1
 fi
 
+# TODO (tenzen-y): Once the changes on https://github.com/kubeflow/testing/pull/974 are reflected in the `public.ecr.aws` registry, we must remove this process.
+# To avoid the `../../../../pkg/mod/k8s.io/client-go@v0.22.2/plugin/pkg/client/auth/exec/metrics.go:21:2: package io/fs is not in GOROOT (/usr/local/go/src/io/fs)` error,
```

Comment:
Is this still needed after https://github.com/kubeflow/testing/pull/974?

Reply:
@johnugeorge Thanks your comments.

Yes, As Jeffwan says,  since `public.ecr.aws/j1r0q0g6/kubeflow-testing:latest` image was manually updated, we can not update its image by merging https://github.com/kubeflow/testing/pull/974.

> However, I forget the image public.ecr.aws/j1r0q0g6/kubeflow-testing:latest was manually updated in the past. 

https://github.com/kubeflow/training-operator/issues/1581#issue-1223868790

Reply:
Ref: https://gallery.ecr.aws/j1r0q0g6/kubeflow-testing (Amazon ECR Public)